enqueue/enqueue-bundle/0.9/config/packages/enqueue.yaml

```diff
@@ -1,4 +1,4 @@
 enqueue:
     default:
-        transport: '%env(ENQUEUE_DSN)%'
+        transport: '%env(resolve:ENQUEUE_DSN)%'
```

Comment:
Can you explain why to use the resolve here?

Reply:
We use `resolve` to be able to look at the DNS in the EnqueueExtension to figure out what transport is used. 

(Im not 100% that this is the case but I assume that's why)

Reply:
Thank you, I was asking because I already use it without the resolve and it works.
Using amqp+lib and sqs

Reply:
This has nothing to do with EnqueueExtension. resolve is used to resolve any parameters found in the DSN, for example `amqp://%amqp_user%:%amqp_pass%@localhost:5672`

Reply:
You are right 👌🏻